test_data/parser/invalid_syntax/missing-semicolon.nix

```diff
@@ -0,0 +1 @@
+{ foo = 1 bar = 2} // { xyz = 3; }
```

Comment:
It's fun seeing how Github syntax highlighting, on the other hand, is _not_ fault-tolerant